charts/trino/templates/deployment-coordinator.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         app: {{ template "trino.name" . }}
         release: {{ .Release.Name }}
         component: coordinator
+        checksum/config: {{ include (print $.Template.BasePath "/configmap-coordinator.yaml") . | sha256sum | trunc 10}}
```

Comment:
Why the `trunc`?

Reply:
The checksum/config value was too large (63 characters is the limit I believe)..but I added `checksum/config` as a child of `spec/template/metadata/annotations` instead of `spec/template/metadata/labels` (according to [helm docs](`spec/template/metadata/annotations`)) and was able to remove `trunc`.